recipes/reports/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "reports" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "72b5d5d1f8a452f4c8b1f69862c2d6bc4ac953490273b23b232dc35692c01755" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: reports-0.3.1.tar.gz
+  url: https://pypi.python.org/packages/24/c0/d8d329f3a5027c3382140e0db925b789eb80615044ce4088d96da529e318/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - easydev
+    - pandas
+    - colormap
+    - jinja2
+
+test:
+  imports:
+    - reports
+    - reports.resources
+
+about:
+  home: 'http://github.com/cokelaer/reports/'
+  license: LGPL-3.0
+  license: LGPL v3+
```

Comment:
The github repo for this project says that it is BSD 3-clause and the `LICENSE` file looks to match this. I just wanted to double check if LGPL was actually correct here?